client/src/theme/styles/components/backend/project-collection/_collection-list.scss

```diff
@@ -41,7 +41,7 @@
   &__inner {
     @include draggable;
     display: flex;
-    align-items: baseline;
+    align-items: center;
```

Comment:
What was the reason for changing this value?

Reply:
If you set `align-items: baseline` the project collection list the project collection name looks just a little higher up than it should be:

![baseline style screenshot](https://user-images.githubusercontent.com/9158517/88966110-a9760980-d260-11ea-9abf-27e1eb73d677.png)

Before I started my work on this, all content in the project-collection-list-items were centered vertically and I wanted to maintain that, so I set the alignment to be centered. It appears that changing the `project-collection-list-item__link` to `display: flex` to limit the length of the span, bumps the text up about 2 pixels and without the centering property, it is about 2 pixels out of place.

Reply:
OK that's fine. You should get your fonts working locally, btw. Using the wrong fonts will make the values in your CSS inaccurate because the glyphs don't have the same shape and size.